streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -470,6 +500,7 @@ public void shouldAddFailedTasksToQueueWhenRestoreThrowsStreamsExceptionWithoutT
         verifyRemovedTasks();
         verifyUpdatingTasks();
         verifyRestoredActiveTasks();
+        verifyNeverCheckpointTasks(task1, task2);
```

Comment:
Got it.

Thinking about it a second time, I think we should just remove this verification then, since with the above fix it's basically the same as our other newly added tests.